src/routes/users.route.js

```diff
@@ -97,4 +98,33 @@ userRouter.get(
   getReadingStats,
 );
 
+/**
+ * @swagger
+ *
+ * /api/v1/users/profiles/:username/stats:
```

Comment:
documentation here should read `username/bookmarks` not `stats`

Reply:
Noted and fixed